src/components/TypingIndicator/TypingIndicator.vue

```diff
@@ -0,0 +1,142 @@
+<!--
+  - @copyright Copyright (c) 2023 Maksim Sukharev <antreesy.web@gmail.com>
+  -
+  - @author Maksim Sukharev <antreesy.web@gmail.com>
+  -
+  - @license GNU AGPL version 3 or any later version
+  -
+  - This program is free software: you can redistribute it and/or modify
+  - it under the terms of the GNU Affero General Public License as
+  - published by the Free Software Foundation, either version 3 of the
+  - License, or (at your option) any later version.
+  -
+  - This program is distributed in the hope that it will be useful,
+  - but WITHOUT ANY WARRANTY; without even the implied warranty of
+  - MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  - GNU Affero General Public License for more details.
+  -
+  - You should have received a copy of the GNU Affero General Public License
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<template>
+	<div v-show="typingParticipants.length" class="indicator">
+		<div class="indicator__wrapper">
+			<div class="indicator__avatars">
+				<AvatarWrapper v-for="(participant, index) in visibleParticipants"
+					:id="participant.actorId"
+					:key="index"
+					:name="participant.displayName"
+					:source="participant.actorType"
+					small
+					condensed
+					:condensed-overlap="8"
+					disable-menu
+					disable-tooltip />
+			</div>
+
+			<p class="indicator__main">
+				<b>{{ visibleParticipantsNamesList }}</b>
+				<span v-if="hiddenParticipantsCount" class="indicator__corresponding">
+					{{ n('spreed', 'and %n other are typing...', 'and %n others are typing...', hiddenParticipantsCount) }}
```

Comment:
Doesn't work for translations

Reply:
Fixed with commit below.

:warning: Please notice that `translate` and `translatePlural` functions escape HTML and sanitize text to prevent XSS attacks by default, so it should be safe to use `v-html` directive. But it's better to be checked: cc @ShGKme 

Link to the library: https://github.com/nextcloud/nextcloud-l10n/blob/1e43c9bd8ba3e814f0a897e86430a77d6cdaacbd/lib/translation.ts#L60-L63